classes/context/Context.php

```diff
@@ -70,6 +70,15 @@ public function areDoisEnabled(): bool
         return (bool) $this->getData(Context::SETTING_ENABLE_DOIS);
     }
 
+    /**
+     * Whether peer reviews are publicly visible for this context
+     *
+     */
+    public function arePeersReviewPublic(): bool
+    {
+        return (bool) $this->getData('enablePublicPeerReviews');
```

Comment:
Could you create a constant above for this string. Something like `SETTING_ENABLE_PUBLIC_REVIEWS`?